bigquery/indexer.py

```diff
@@ -269,7 +233,14 @@ def index_fields(es, index_name, table, sample_id_column):
             id_prefix = "samples." + id_prefix
 
     field_docs = _field_docs_by_id(id_prefix, '', fields)
-    indexer_util.bulk_index_docs(es, index_name, field_docs)
+    field_docs_dict = {}
+    for field_doc_id, field_doc in field_docs:
+        if field_doc_id in field_docs_dict:
+            field_docs_dict[field_doc_id].update(field_doc)
+        else:
+            field_docs_dict[field_doc_id] = field_doc
+    indexer_util.bulk_index_docs(
+        es, index_name, field_docs_dict, op_type='update')
```

Comment:
For simplicity change let's just create here. Also, conceptually it should be create -- each doc is only written once. Delete bulk_index_docs op_type parameter.